### PR TITLE
Indent after umcompleted condition

### PR DESCRIPTION
**Pull request by yuutayamada**
Hi, I added a feature to indent if the condition parts are multiple lines.

For example:
```nim
if longCondition and
   longCondition2: # <- I mean this line's indent
  echo("foo")
```

By default, I set 4 spaces, which is Araq usually does.
(see also [this discussion](http://forum.nim-lang.org/t/1507))

### CODE REVIEW

File: nim-helper.el [hunk 1]
```diff
@@ -989,5 +989,16 @@ default to utf-8."
   (or (nim-info-encoding-from-cookie)
       'utf-8))
 
+(defun nim-helper-line-contain-p (char &optional pos)
+  "Return non-nil if the current line has CHAR.
+But, string's CHAR is ignored.  If you set POS the check starts from POS."
```
Comment (reactormonk) [inline nim-helper.el]: Which string's?

**Comment by reactormonk** [comment]
Does it work with 

```nim
when "Even if string's :" == "is exist" and
# comment here
"The : should not affect" == "indent result":
echo "foo"
```

File: nim-helper.el (hunk 1 above)
Comment (yuutayamada) [inline nim-helper.el]: Sorry, I should have written string-face's character

**Comment by yuutayamada** [comment]
> Does it work with ...

Thanks, I didn't imagine that case. Fortunately It indented even if there is inside comment.
(I checked by hand)

**Comment by yuutayamada** [comment]
Oh, sorry, currently indent-region function ignore comment, but C-i(tab) key works...

**Comment by yuutayamada** [comment]
In nim-indent-region function, following condition was the problem. 
(I changed this style at d94b9b0)

```lisp
(and
  (nim-info-current-line-comment-p -1)
  ;; Unless this line is a comment too.
  (not (nim-info-current-line-comment-p))))
```

If I omit this condition, I could pass all tests and your example.
But, I'm still not sure this is ok or not. If you ok, I would remove above condition part
and I would rethink about this condition when a problem is happened.
(Originally I imported this logic from python.el, so it might have some insight, but
currently I don't know...)

**Comment by reactormonk** [comment]
If it passes the tests, it's good enough, until someone brings another case where it doesn't work. My understanding of the indent code is rather limited.

**Comment by yuutayamada** [comment]
Thanks. I updated. (I added the comment test as well)

**Pull request merged**